client/src/components/Hamburger/Hamburger.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
+import './Hamburger.css';
+
+library.add(faBars, faTimes)
+
+class Hamburger extends Component {
+    constructor(props) {
+        super()
+        this.state = {
+            open: false,
+            iconList: "bars",
+            typeName: "Bars"
+        }
+        this.openMenu = this.openMenu.bind(this)
+    }
+    // let iconList = "bars"
+    // let classList = "";
+    // let classList = "hamburger";
+    
+    // let iconList = `${props.icon}`;
+    
+    // let types = "icon";
+    // let types = "icon";
+    // if (types.includes(props.type)) {
+    //     classList += ` hamburger-${props.type}`
+    // }
+    openMenu() {
+        this.setState({
+        ...this.state,
+        iconList: "times",
+        open: !this.state.open})
+        console.log(this.state.iconList + " " + this.state.classList);
```

Comment:
Can you remove any `console.log` statements?

Reply:
@ZakkMan I will remove all console.log codes once I get everything to function properly but for the moment I'm using it to see if any objects change during my click events